client/inbox-panel/index.js

```diff
@@ -234,6 +246,45 @@ const InboxPanel = () => {
 		}
 	};
 
+	const dismissAllNotes = async () => {
```

Comment:
Can this method be moved into `DismissAllModal`?

Reply:
That's a good idea. Updated in https://github.com/woocommerce/woocommerce-admin/pull/7871/commits/da528a7663d316b0d256c5f5c50435f942399b22